maintainers/maintainer-list.nix

```diff
@@ -9452,6 +9452,12 @@
     githubId = 6789916;
     name = "Jason Odoom";
   };
+  jatcwang = {
+    email = "jatcwang@gmail.com";
+    github = "jatcwang";
+    githubId = 4957161;
+    name = "Jacob Wang";
+  };
```

Comment:
This is normally done as its own commit titled`maintainers: add jatcwang`

Reply:
done